.travis.yml

```diff
@@ -1,40 +1,35 @@
 language: python
 python:
-  - 2.7
-  - 3.4
-  - 3.5
-  - 3.6
+- 2.7
+- 3.6
 os:
 - linux
-matrix:
-  allow_failures:
-    - os: osx
```

Comment:
Just a question while I see this: would the conda installation in travis allow us to run on osx too ?

Reply:
Yes, I can try adding that and see what happens.